app/ai/voice/agents/breeze_buddy/utils/playground.py

```
@@ -1,30 +1,68 @@
+                    # Re-render role_messages
+                    for msg in node.get("role_messages", []):
+                        if isinstance(msg, dict) and "content" in msg:
+                            for key, value in template_vars.items():
+                                msg["content"] = msg["content"].replace(
+                                    f"{{{key}}}", str(value)
+                                )
+                logger.info(
+                    f"Re-rendered messages after flow override for lead {lead.id}"
+                )
+
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)